network/server/tchannelthrift/cluster/service.go

```diff
@@ -180,7 +180,7 @@ func (s *service) Truncate(tctx thrift.Context, req *rpc.TruncateRequest) (*rpc.
 	if !ok {
 		return nil, tterrors.NewInternalError(errors.New("unable to get an admin session"))
 	}
-	truncated, err := adminSession.Truncate(ts.BinaryID(req.NameSpace))
+	truncated, err := adminSession.Truncate(ts.StringID(string(req.NameSpace)))
```

Comment:
Don't have to, but it's either this or creating a new `checked.NewBytes(...)`.  I'll change to `checked.NewBytes(...)` and use `ts.BinaryID(...)`, but it really is quite similar - especially for truncate which barely if ever gets called.